db/seeds/baseline_rates/linehauls.rb

```diff
@@ -0,0 +1,46 @@
+module Seeds
+  module BaselineRates
+    class Domestic400NG
```

Comment:
Same question here as above. Are these `baseline_rates/foo.rb` seeds files sub-classes of `Domestic400NG` or something similar?

Reply:
I'm using monkey patching for a rare acceptable purpose - to organize the functionality of one class in multiple files in a logical way. (This technique is common in some other languages; in C#, this would be a [partial class](http://www.tutorialsteacher.com/csharp/csharp-partial-class).) Each file in the baseline_rates folder (other than the main domestic_400ng.rb) contains parsing code for just one Sheet of the 400NG xlsx file. If I put it all in one file, it would make no difference to the functionality, but would probably make CodeClimate think the file is too big.

I decided to make it one class because it's all part of 400 NG parsing - the Additional Rates, Geographic Schedules, and Linehaul parsers make no sense outside of that context.